pages/pet-detail-page/[id].js

```diff
@@ -100,12 +184,48 @@ export default function PetDetailPage({ myPets, onGameUpdate, onSetIsDead }) {
         <StatusBar text={"Hunger"} value={currentPet.hunger} />
         <StatusBar text={"Happiness"} value={currentPet.happiness} />
         <StatusBar text={"Energy"} value={currentPet.energy} />
-        <StyledPetImage
-          src={isDead ? graveImage : image}
-          alt={type}
-          height={150}
-          width={150}
-        />
+
+        <button
+          onClick={() => handleFeed(10)}
+          disabled={isInteracting.duration > 0}
+        >
+          <Image alt="Hunger" src={hungerImage} width={50} height={50}></Image>
```

Comment:
It's still working but I am sure that <Image /> is supposed to be a self closing tag.

Reply:
... the other 2 Images as well :)